content/dstu2/document-reference.md

```diff
@@ -110,7 +110,48 @@ present.
    x-xss-protection → 1; mode=block
 </pre>
 
+## Operation: docref
+
+Argonaut operation for querying DocumentReferences for the supplied parameters:
+
+    GET /DocumentReference/$docref?:parameters
+
+_Implementation Notes_
+
+* The [DocumentReference.relatesTo] modifier element is not supported and will not be returned.
+
+### Terminology Bindings
+
+<%= terminology_table(:document_reference_docref, :dstu2) %>
+
+### Parameters
+
+ Name     | Required? | Type          | Description
+----------|-----------|---------------|-------------------------------------------------
+`patient` | Yes       | [`reference`] | A reference to the patient whose document references are required. Example: `14067892`
+`type`    | Yes       | [`token`]     | The document reference type, can be a list of comma separated values. Example: `http://loinc.org|34133-9`
+`start`   | No        | [`date`]      | The start of the date range form which document reference records are required. If not provided, then all records upto the end or current date are considered. Example: `2014-09-24T12:00:00.000Z`
```

Comment:
The start of the date range -form- *from* which document reference records -are required- (should be loaded?). If not provided, -then- all records up (add space) to the end or current date are considered.

Do we want to say something like "If not provided, records will be loaded from the beginning of time." instead of the current second sentence?